models/lifi/fantom/lifi_v2_fantom_trades.sql

```diff
@@ -67,31 +69,31 @@ SELECT
     end as token_pair,
     dexs.token_bought_amount_raw / power(10, erc20a.decimals) AS token_bought_amount,
     dexs.token_sold_amount_raw / power(10, erc20b.decimals) AS token_sold_amount,
-    CAST(dexs.token_bought_amount_raw AS DECIMAL(38,0)) AS token_bought_amount_raw,
-    CAST(dexs.token_sold_amount_raw AS DECIMAL(38,0)) AS token_sold_amount_raw,
+    CAST(dexs.token_bought_amount_raw AS UINT256) AS token_bought_amount_raw,
+    CAST(dexs.token_sold_amount_raw AS UINT256) AS token_sold_amount_raw,
```

Comment:
these are both already `uint256` at the source, we can remove the cast

Reply:
will fix, there are a couple of other changes @antonio-mendes mentioned in other models as well... If you're reviewing/merging this evening I can submit the fix for all the models since it won't take a lot of time.. actually lemme get to it now.. I was trying to complete something else but It should be good.

Reply:
cleaned up will fix other models now @jeff-dude